run/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ endforeach (_file RANGE 0 ${_length})
 # perform some checks on kinematics and track references
 if (HAVESIMULATION)
   add_test(NAME o2sim_G4 COMMAND ${CMAKE_BINARY_DIR}/bin/o2sim -n 2 -m PIPE ITS TPC TRD TOF EMC PHS -e TGeant4)
+  add_test(NAME o2sim_G4_mt COMMAND ${CMAKE_BINARY_DIR}/bin/o2sim -n 10 -m PIPE ITS TPC TRD TOF EMC PHS -e TGeant4 --isMT on)
```

Comment:
I believe that one still needs to define test properties such as "acceptance criteria" etc. You can take a look how this is done for the other tests.